src/architecture/clients.md

```diff
@@ -1,193 +1,194 @@
+  блокчейна. По сути, с точки зрения пользователя, такой блокчейн ничем не
+  отличается от распределенной базы данных.
+
+Одним из способов устранения недостатков "аудита выделенными узлами аудиторами"
+является внедрение *легких клиентов*, также известных как *тонкие клиенты* или
+просто *клиенты*. В Биткоин блокчейне, эти клиенты также известны как клиенты
+[SPV (simple payment verification)](https://en.bitcoin.it/wiki/Thin_Client_Security).
+Легкие клиенты - это программы, способные копировать и проверять небольшую часть
+информации, хранящейся в блокчейне. Обычно клиенты проверяют информацию,
+относящуюся к конкретному пользователю (например, историю его транзакций). Эта
+проверка возможна благодаря использованию специфических контейнеров данных в
+блокчейне: индексов [Merkle][mt-index] и [Merkle Patricia][mpt-index].
+
+Преимуществами этого подхода являются:
+
+- **Снижение необходимости доверия третьим сторонам**: проверка всех данных,
+  возвращаемых блокчейном в ответ на запросы, выполняется самим пользователем
+  на его машине
+- Постоянный (хотя и частичный) аудит системы возможен без необходимости
+  использования вычислительных ресурсов, сопоставимых с ресурсами валидаторов с точки
+  зрения производительности. Проверяются только релевантные данные
+- Чтобы начать или возобновить аудит системы, не требуется время на синхронизацию
+
+!!! note "Примечание"
+    Пользователю все еще нужно доверять разработчикам легкого клиента, если
+    клиент имеет закрытый исходный код. Если же исходный код открыт, пользователь
+    должен выполнить аудит кода легкого клиента, чтобы полностью исключить
+    необходимость доверия третьим сторонам.
+
+Наличие легких клиентов не означает отсутствие узлов-аудиторов, поскольку их
+задачи различны. Легкие клиенты проверяют данные конкретного пользователя, в
+то время как узлы-аудиторы проверяют блокчейн в целом.
+
+Безопасность легких клиентов сравнима с проверками безопасности [TLS][wiki:tls],
+встроенными в веб-браузеры. Это не является прямой заменой аудиту, выполняемому
+узлами-аудиторами, но обеспечивает измеримую степень защиты от
+[атак посредников][wiki:mitm] и злонамеренно действующих узлов, с которыми может
+взаимодействовать клиент. В то же время, если легкие клиенты охватывают все
+транзакции блокчейна, их *коллективная* безопасность может стать сопоставимой с
+безопасностью, предоставляемой узлами-аудиторами.
+
+Наличие легких клиентов в блокчейн системе приводит к определенным трудностям в
+процессе разработки:
+
+- Бэкэнд разработчики должны быть в согласии с разработчиками клиентов в
+  вопросах API запросов и форматов криптографических доказательств (фактически,
+  модели данных блокчейна)
+- Любые изменения в модели данных блокчейна должны сопровождаться
+  соответствующими изменениями в логике проверки доказательств, выполняемой
+  легкими клиентами
+- Поскольку легкий клиент существенно расширяет доступ к Exonum REST конечным
```

Comment:
> Поскольку легкий клиент существенно расширяет доступ к Exonum REST конечным точкам с криптографией

Предложение совершенно непонятно. Прежде всего, "конечным точкам Exonum". @slowli какой смысл у этого предложения? (в оригинале Since the light client substantially expands an access to Exonum REST endpoints with cryptography, it may be necessary to create multiple light clients and continuously support their codebase)

Reply:
Имхо, это предложение можно удалить; см. #215.